src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -761,22 +762,26 @@ export class GwtCallback extends EventEmitter {
     });
 
     ipcMain.on('desktop_set_tutorial_url', (event, url) => {
-      GwtCallback.unimpl('desktop_set_tutorial_url');
+      this.getSender('desktop_set_tutorial_url', event.processId, event.frameId).setTutorialUrl(url);
     });
 
     ipcMain.on('desktop_set_viewer_url', (event, url) => {
       this.getSender('desktop_set_viewer_url', event.processId, event.frameId).setViewerUrl(url);
     });
 
-    ipcMain.on('desktop_reload_viewer_zoom_window', (event, url) => {
+    ipcMain.on('desktop_reload_viewer_zoom_window', (_event, url) => {
```

Comment:
is this our convention to use _event if it isn't being called? I haven't been doing that

Reply:
Yes, `eslint` will complain if there is an unused parameter. The underscore signals that it's there, because of argument order, but unused. It's perfectly fine without it but addresses the linter warning (I think it's configured as a warning).